Code/GraphMol/MolDraw2D/MolDraw2DSVG.cpp

```diff
@@ -20,17 +20,21 @@ std::string DrawColourToSVG(const DrawColour &col) {
   const char *convert = "0123456789ABCDEF";
   std::string res(7, ' ');
   res[0] = '#';
-  unsigned int v;
+  int raw_color;
+  unsigned int clamped_color;
   unsigned int i = 1;
-  v = int(255 * col.get<0>());
-  res[i++] = convert[v / 16];
-  res[i++] = convert[v % 16];
-  v = int(255 * col.get<1>());
-  res[i++] = convert[v / 16];
-  res[i++] = convert[v % 16];
-  v = int(255 * col.get<2>());
-  res[i++] = convert[v / 16];
-  res[i++] = convert[v % 16];
+  raw_color = int(255 * col.get<0>());
```

Comment:
Nice catch, could also use.   I suppose I would prefer to use rdcast if we actually want to catch the errors in debug mode.

```
unsigned int v = rdcast<unsigned char>( 255 * col.get<0>() );
```

In debug modes, this will throw an exception and allow us to track down the errors.  In release, it will simply clamp.